app.js

```diff
@@ -0,0 +1,25 @@
+console.log('It works!!');
+
+
+
+//==============================================
+
+
+
+const img = document.createElement('img');
+img.setAttribute('src','https://source.unsplash.com/random');
```

Comment:
you can make this part of the callback function.

you can also write this code so you can get a radndom picture of a spacific size

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```